layersvt/device_simulation_examples/sdk_sample_configs/minimum_vulkan_1_2.json

```diff
@@ -0,0 +1,1133 @@
+        "shaderResourceResidency": 0,
+        "shaderResourceMinLod": 0,
+        "sparseBinding": 0,
+        "sparseResidencyBuffer": 0,
+        "sparseResidencyImage2D": 0,
+        "sparseResidencyImage3D": 0,
+        "sparseResidency2Samples": 0,
+        "sparseResidency4Samples": 0,
+        "sparseResidency8Samples": 0,
+        "sparseResidency16Samples": 0,
+        "sparseResidencyAliased": 0,
+        "variableMultisampleRate": 0,
+        "inheritedQueries": 0
+    },
+    "VkPhysicalDevice16BitStorageFeatures": {
+        "storageBuffer16BitAccess": 1,
+        "uniformAndStorageBuffer16BitAccess": 0,
+        "storagePushConstant16": 0,
+        "storageInputOutput16": 0
+    },
+    "VkPhysicalDevice8BitStorageFeatures": {
+        "storageBuffer8BitAccess": 0,
+        "uniformAndStorageBuffer8BitAccess": 0,
+        "storagePushConstant8": 0
+    },
+    "VkPhysicalDeviceDescriptorIndexingFeatures": {
+        "shaderInputAttachmentArrayDynamicIndexing": 0,
+        "shaderUniformTexelBufferArrayDynamicIndexing": 0,
+        "shaderStorageTexelBufferArrayDynamicIndexing": 0,
+        "shaderUniformBufferArrayNonUniformIndexing": 0,
+        "shaderSampledImageArrayNonUniformIndexing": 0,
+        "shaderStorageBufferArrayNonUniformIndexing": 0,
+        "shaderStorageImageArrayNonUniformIndexing": 0,
+        "shaderInputAttachmentArrayNonUniformIndexing": 0,
+        "shaderUniformTexelBufferArrayNonUniformIndexing": 0,
+        "shaderStorageTexelBufferArrayNonUniformIndexing": 0,
+        "descriptorBindingUniformBufferUpdateAfterBind": 0,
+        "descriptorBindingSampledImageUpdateAfterBind": 0,
+        "descriptorBindingStorageImageUpdateAfterBind": 0,
+        "descriptorBindingStorageBufferUpdateAfterBind": 0,
+        "descriptorBindingUniformTexelBufferUpdateAfterBind": 0,
+        "descriptorBindingStorageTexelBufferUpdateAfterBind": 0,
+        "descriptorBindingUpdateUnusedWhilePending": 0,
+        "descriptorBindingPartiallyBound": 0,
+        "descriptorBindingVariableDescriptorCount": 0,
+        "runtimeDescriptorArray": 0
+    },
+    "VkPhysicalDeviceHostQueryResetFeatures": {
+        "hostQueryReset": 0
+    },
+    "VkPhysicalDeviceImagelessFramebufferFeatures": {
+        "imagelessFramebuffer": 0
+    },
+    "VkPhysicalDeviceMultiviewFeatures": {
+        "multiview": 1,
+        "multiviewGeometryShader": 0,
+        "multiviewTessellationShader": 0
+    },
+    "VkPhysicalDeviceProtectedMemoryFeatures": {
+        "protectedMemory": 0
+    },
+    "VkPhysicalDeviceSamplerYcbcrConversionFeatures": {
+        "samplerYcbcrConversion": 0
+    },
+    "VkPhysicalDeviceScalarBlockLayoutFeatures": {
+        "scalarBlockLayout": 0
+    },
+    "VkPhysicalDeviceSeparateDepthStencilLayoutsFeatures": {
+        "separateDepthStencilLayouts": 0
+    },
+    "VkPhysicalDeviceShaderAtomicInt64Features": {
+        "shaderBufferInt64Atomics": 1,
+        "shaderSharedInt64Atomics": 1
```

Comment:
This is supported by a limited set of devices.

Reply:
https://vulkan.gpuinfo.org/listdevicescoverage.php?core=1.2&feature=shaderBufferInt64Atomics&platform=windows&option=not